packages/bpk-component-rating/src/BpkRating.js

```diff
@@ -72,10 +74,18 @@ const BpkRating = (props: Props) => {
         <strong>{adjustedValue}</strong>
       </BpkText>
       <div className={getClassName('bpk-rating__text')}>
-        <BpkText textStyle="base" tagName="span" aria-hidden="true">
+        <BpkText
+          textStyle={RATING_SIZES[size]}
+          tagName="span"
+          aria-hidden="true"
+        >
           <strong>{title}</strong>
         </BpkText>
-        <BpkText textStyle="xs" tagName="span" aria-hidden="true">
+        <BpkText
+          textStyle={size === RATING_SIZES.lg ? 'base' : 'sm'}
```

Comment:
Previously, `textStyle` was `xs` but now it's `base` or `sm`. How come it's changed?

Reply:
It is due to `xs` and `sm` being the same values so didn't make a difference so thought it would fit the sizings better rather than missing `sm`

Reply:
Ah ok 👍